Tools/build/smelly.py

```diff
@@ -11,6 +11,11 @@
 if sys.platform == 'darwin':
     ALLOWED_PREFIXES += ('__Py',)
 
+# mimalloc doesn't use static, but it's symbols are not exported
+# from the shared library.  They do show up in the static library
+# before its linked into an executable.
+ALLOWED_STATIC_PREFIXES = ('mi_', '_mi_')
```

Comment:
Thank you for distinguishing local and exported symbols. It's good that "mi" and "_mi_" prefixes are only allowed for local symbols.